Build error on latest released version

I added `gocodewalker` to my project and got a build error:

```shell
$ go get github.com/boyter/gocodewalker
$ go build .
# github.com/boyter/gocodewalker
../../../go/pkg/mod/github.com/boyter/gocodewalker@v1.2.0/file.go:294:23: cannot use file (variable of type fs.FileInfo) as fs.DirEntry value in argument to IsHidden: fs.FileInfo does not implement fs.DirEntry (missing method Info)
../../../go/pkg/mod/github.com/boyter/gocodewalker@v1.2.0/file.go:431:23: cannot use dir (variable of type fs.FileInfo) as fs.DirEntry value in argument to IsHidden: fs.FileInfo does not implement fs.DirEntry (missing method Info)
```

Downgrading to `v1.1.1` works:

```shell
$ go get github.com/boyter/gocodewalker@v1.1.1
go: downgraded github.com/boyter/gocodewalker v1.2.0 => v1.1.1
$ go build .
```

Looks like there's a disparity between [pkg.go.dev](https://pkg.go.dev/github.com/boyter/gocodewalker?tab=versions) and your tags. Perhaps the `v1.2.0` tag was deleted? If so, you'll need to add [a `retract` line to your go.mod file](https://stackoverflow.com/questions/68599703/how-do-i-delete-a-package-that-i-accidentally-published-to-go-dev) and push a new version to prevent `v1.2.0` from being downloaded.

--- a/go.mod
+++ b/go.mod
@@ -3,3 +3,5 @@ module github.com/boyter/gocodewalker
 go 1.20
 
 require github.com/danwakefield/fnmatch v0.0.0-20160403171240-cbb64ac3d964
+
+retract v1.2.0

